packages/remediations/src/helpers/host-helper.js

```diff
@@ -0,0 +1,9 @@
+import instance from '@redhat-cloud-services/frontend-components-utilities/interceptors';
```

Comment:
Duplicate file, it's the same as `packages/remediations/src/api/inventory.js`

Reply:
Oh, good catch, thanks for reporting :-)